contents/handbook/people/team-structure/design.md

```diff
@@ -53,6 +53,8 @@ TL;DR: If your team doesn't have a dedicated designer, create an Issue and add i
 1. What do you need designed and why?
 2. What is the deadline?
 
+If your design request is for a blog post post a card with the same information within the [Art Requests](https://github.com/orgs/PostHog/projects/14) board, as well as adding it to the [Content Calendar](https://docs.google.com/spreadsheets/d/1-6QYxi46d5y88BQ8vdGWmgrFZBbCMs1CAIc5JGLuf4Y/edit#gid=482712771) - this helps the Design board and Lotties brain to stay uncluttered. 
```

Comment:
Flyby suggestion and I'm sticking my nose in, but would this be a clearer explanation? 

If your request **is** for a blog post or social media, create an issue in the relevant repo and add it to the [Art Requests](https://github.com/orgs/PostHog/projects/14) project, as well as the [Content Calendar](https://docs.google.com/spreadsheets/d/1-6QYxi46d5y88BQ8vdGWmgrFZBbCMs1CAIc5JGLuf4Y/edit#gid=482712771). 

If your request **isn't** for a blog post or social media, create an issue in the relevant repo and add it to the [Design project](https://github.com/orgs/PostHog/projects/3). 

In either case your issue doesn’t have to be isolated to the request, but you should include the following details so we can prioritize accordingly:

1. What do you need designed and why?
2. When is the deadline?

If there isn’t a clear deadline, we’ll prioritize according to our availability.